content/docs/tools/teddyCloud/_index.md

```diff
@@ -31,10 +31,15 @@ Please check wether you got a Toniebox with a CC3200, CC3235 or ESP32 and contin
 Please put your content into the ```/data/content/default/``` in the same structure as on your toniebox. You can edit ```500304E0.json``` file beside the content files to mark them as live or you can prevent the usage of the Boxine cloud for that tag with the nocloud parameter. By setting a source teddyCloud can stream any content that ffmpeg can decode (urls and files).
 
 ### Webinterface
-Currently the interface to teddycloud is reachable through the IP of the docker container at port 80 or 443 (depending on your ```docker-compose.yaml```). Changes affecting the toniebox (volume, LED) which are made through this interface will only be reflected onto the toniebox after pressing the big ear for a few seconds until a beep occurs.
+Currently the interface to teddycloud is reachable through the IP of the docker container at port 80 or 443 (depending on your ```docker-compose.yaml```). Changes affecting the toniebox (volume, LED) which are made through this interface will only be reflected onto the toniebox after pressing the big ear for a few seconds until a beep occurs. 
+
+![image](https://github.com/toniebox-reverse-engineering/toniebox-reverse-engineering.github.io/assets/5244525/344c936d-9d7f-4bf4-80b7-57f8be634ec3)
```

Comment:
Please save the image into:
https://github.com/toniebox-reverse-engineering/toniebox-reverse-engineering.github.io/tree/master/static/img

And embed it this way:
https://github.com/toniebox-reverse-engineering/toniebox-reverse-engineering.github.io/blob/master/content/docs/custom-firmware/cc3200/hackieboxng-bl/bootloader.md?plain=1

Reply:
just did the change